khal/cli.py

```diff
@@ -230,7 +230,8 @@ def calendar(ctx, days, events, dates):
             weeknumber=ctx.obj['conf']['locale']['weeknumbers'],
             show_all_days=ctx.obj['conf']['default']['show_all_days'],
             days=days or ctx.obj['conf']['default']['days'],
-            events=events
+            events=events,
+            conf=ctx.obj['conf']
```

Comment:
Please decide on one strategy:

* Pass each piece of data explicitly (firstweekday, encoding, locale, ...)
* Pass the config as single param

Since the choice has already been made, please try to stick with it.